sentinelone/help.md

```diff
@@ -125,8 +125,6 @@ Example input:
 
 ```
 {
-  "blacklist_state": true,
-  "description": "Hash Blacklisted from InsightConnect",
```

Comment:
@r7-kszczepanskagorna I think this got removed by the tooling, can we add it back?

Reply:
@jschipp-r7 I remove it because this two inputs are not in plugin.spec (whole inputs, not only example field). 
FYI - In this plugin are a lot of missing example:
```
In action "blacklist_by_ioc_hash": input "agent_id", there is no example field.
In action "create_ioc_threat": input "hash", there is no example field.
In action "create_ioc_threat": input "agent_id", there is no example field.
In action "create_ioc_threat": input "annotation", there is no example field.
In action "create_ioc_threat": input "annotation_url", there is no example field.
In action "create_ioc_threat": input "group_id", there is no example field.
In action "create_ioc_threat": input "path", there is no example field.
In action "mitigate_threat": input "threat_id", there is no example field.
In action "mitigate_threat": input "action", there is no example field.
In action "mark_as_benign": input "threat_id", there is no example field.
In action "mark_as_benign": input "target_scope", there is no example field.
In action "mark_as_benign": input "whitening_option", there is no example field.
In action "mark_as_threat": input "threat_id", there is no example field.
In action "mark_as_threat": input "target_scope", there is no example field.
In action "mark_as_threat": input "whitening_option", there is no example field.
In action "activities_list": input "group_ids", there is no example field.
In action "activities_list": input "include_hidden", there is no example field.
In action "activities_list": input "skip", there is no example field.
In action "activities_list": input "site_ids", there is no example field.
In action "activities_list": input "agent_ids", there is no example field.
In action "activities_list": input "skip_count", there is no example field.
In action "activities_list": input "ids", there is no example field.
In action "activities_list": input "created_at_lt", there is no example field.
In action "activities_list": input "created_at_lte", there is no example field.
In action "activities_list": input "created_at_gt", there is no example field.
In action "activities_list": input "created_at_gte", there is no example field.
In action "activities_list": input "created_at_between", there is no example field.
In action "activities_list": input "cursor", there is no example field.
In action "activities_list": input "count_only", there is no example field.
In action "activities_list": input "account_ids", there is no example field.
In action "activities_list": input "limit", there is no example field.
In action "activities_list": input "sort_by", there is no example field.
In action "activities_list": input "activity_types", there is no example field.
In action "activities_list": input "threat_ids", there is no example field.
In action "activities_list": input "sort_order", there is no example field.
In action "activities_list": input "user_emails", there is no example field.
In action "activities_list": input "user_ids", there is no example field.
In action "apps_by_agent_ids": input "ids", there is no example field.
In action "agents_summary": input "site_ids", there is no example field.
In action "agents_summary": input "account_ids", there is no example field.
In action "name_available": input "name", there is no example field.
In action "agents_decommission": input "filter", there is no example field.
In action "agents_disconnect": input "filter", there is no example field.
In action "agents_processes": input "ids", there is no example field.
In action "agents_initiate": input "filter", there is no example field.
In action "agents_uninstall": input "filter", there is no example field.
In action "agents_shutdown": input "filter", there is no example field.
In action "agents_restart": input "filter", there is no example field.
In action "agents_reload": input "filter", there is no example field.
In action "agents_reload": input "module", there is no example field.
In action "agents_fetch_logs": input "filter", there is no example field.
In action "agents_abort_scan": input "filter", there is no example field.
In action "agents_connect": input "filter", there is no example field.
In trigger "get_threats": input "resolved", there is no example field.
In trigger "get_threats": input "classifications", there is no example field.
In trigger "get_threats": input "agent_is_active", there is no example field.
In trigger "get_threats": input "engines", there is no example field.
In trigger "get_threats": input "frequency", there is no example field.
```
We can fix it in other ticket if you want :)